rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -6913,6 +6914,113 @@ void getEstimatedFeesForNextPegOutEvent_zero_pegouts() throws IOException {
         assertEquals(Coin.ZERO, bridgeSupport.getEstimatedFeesForNextPegOutEvent());
     }
 
+    private static Stream<Arguments> fedProvider() {
+        BridgeRegTestConstants bridgeConstantsRegtest = BridgeRegTestConstants.getInstance();
+        List<FederationMember> members = FederationMember.getFederationMembersFromKeys(
+            PegTestUtils.createRandomBtcECKeys(7)
+        );
+
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP271)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP385)).thenReturn(true);
+
+        return Stream.of(
+            Arguments.of(activations, bridgeConstantsRegtest.getGenesisFederation()),
+            Arguments.of(activations, new P2shErpFederation(
+                members,
+                Instant.now(),
+                1L,
+                bridgeConstantsRegtest.getBtcParams(),
+                bridgeConstantsRegtest.getErpFedPubKeysList(),
+                bridgeConstantsRegtest.getErpFedActivationDelay(),
+                activations
+            ))
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("fedProvider")
+    void getEstimatedFeesForNextPegOutEvent_zero_pegouts_after_RSKIP385(ActivationConfig.ForBlock activations, Federation federation) throws IOException {
```

Comment:
All tests repeat the same logic, only changes the amount of pegouts and the expected value. Can this be parameterized as well?

Reply:
Refactored done.